src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -392,5 +398,10 @@ trait WorkflowComponent {
         findWorkflowMessagesById(id).delete andThen
           findInputResolutionsByWorkflowId(id).delete
     }
+
+    def deleteFailureResolutions(workflowFailureId: Long): DBIOAction[Int, NoStream, Read with Write] = {
+      deleteWorkflowFailureAttributes(workflowFailureId) andThen
```

Comment:
Oh, Biden just pointed out that a failure will cause the entire txn to be rolled back anyway. So the difference is moot here.